deploy/helm/pulumi-operator/Chart.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v2
+name: pulumi-kubernetes-operator
+description: A Helm chart for the Pulumi Kubernetes Operator
+home: https://pulumi.com
+sources:
+  - https://github.com/pulumi/pulumi-kubernetes-operator
+
+icon: https://www.pulumi.com/logos/brand/twitter-card.png
+
+type: application
+
+version: 0.1.0
+appVersion: "1.10.1"
```

Comment:
We do a tiny bit of a dance in the Makefile to render the right version etc. at release time for the existing kubernetes configs etc.: https://github.com/pulumi/pulumi-kubernetes-operator/blob/master/Makefile#L60-L70 which simplifies having to update versions etc. in places. Adding a similar task for the chart should take care of it?